pbiapi/pbiapi.py

```diff
@@ -319,15 +320,15 @@ def import_file_into_workspace(self, workspace_name, skip_report, file_path, dis
                 self.force_raise_http_error(response)
 
             if response.json()["importState"] == "Succeeded":
-                print("Import complete")
+                logging.info("Import complete")
                 return
             else:
-                print("Import in progress...")
+                logging.info("Import in progress...")
 
     @staticmethod
     def force_raise_http_error(
         response: requests.Response, expected_codes: Union[List[int], int] = HTTP_OK_CODE
     ) -> NoReturn:
-        print(f"Expected response code(s) {expected_codes}, got {response.status_code}: {response.text}.")
+        logging.exception(f"Expected response code(s) {expected_codes}, got {response.status_code}: {response.text}.")
```

Comment:
I'm not sure this will work - think `logging.error` should be used here.

Reply:
✅